inductiva/simulation/simulator.py

```diff
@@ -16,10 +16,27 @@ class Simulator(ABC):
     def api_method_name(self) -> str:
         pass
 
+    def _setup_dirs(self, input_dir: types.Path,
```

Comment:
Call this `_setup_io_dirs` or `_setup_input_output_dirs` to make it explicit that the directories being created are for input/output.